vllm/v1/worker/tpu_model_runner.py

```diff
@@ -53,12 +53,11 @@
 
 logger = init_logger(__name__)
 
-# Here we utilize the behavior that out-of-bound index is ignored.
-# FIXME(woosuk): Find a more reliable way to prevent possible bugs.
-_PAD_SLOT_ID = 1_000_000_000
 INVALID_TOKEN_ID = -1
 # Smallest output size
 MIN_NUM_SEQS = 8
+# Block size used for kv cache updating kernel
+KV_CACHE_UPDATE_BLOCK_SIZE = 8
```

Comment:
It would be good to specify what dim this is blocking. For instance, why isn't it the same as the kv cache block size? What is the reason for not tuning this higher?

Reply:
I changed it to `NUM_SLICES_PER_KV_CACHE_UPDATE_BLOCK` already.